tests/examples/test_dataclass_example.py

```diff
@@ -159,7 +159,7 @@ def test_nesting() -> None:
   name: manager
   height: TALL
 """
-    assert conf.pretty() == expected
+    assert OmegaConf.to_yaml(conf) == expected
```

Comment:
after you deprecate, add another test validating the deprecation behavior (see other similar tests).

Reply:
Gotta leave, I'll add documentation and change and change the jupyter notebook.

Reply:
Do I move test_nesting too ? I testes more than to_yaml.

Reply:
no.